other/pascal_triangle.py

```diff
@@ -0,0 +1,69 @@
+"""
+wissamfawaz12@gmail.com | github.com/wissamfawaz
+This implementation demonstrates how to generate the
+elements of a Pascal's triangle.
+What is Pascal's triangle?
+- Refer to (https://en.wikipedia.org/wiki/Pascal%27s_triangle)
+for more info about this triangle.
+"""
+
+
+def generate_pascal_triangle(num_rows: int) -> list[list[int]]:
+    """
+    Print Pascal's triangle for different number of rows
+    >>> generate_pascal_triangle(1)
+    [[1]]
+    >>> generate_pascal_triangle(2)
+    [[1], [1, 1]]
+    >>> generate_pascal_triangle(3)
+    [[1], [1, 1], [1, 2, 1]]
+    >>> generate_pascal_triangle(4)
+    [[1], [1, 1], [1, 2, 1], [1, 3, 3, 1]]
+    >>> generate_pascal_triangle(5)
+    [[1], [1, 1], [1, 2, 1], [1, 3, 3, 1], [1, 4, 6, 4, 1]]
+    """
+    triangle: list[list[int]] = []
+    for current_row_idx in range(num_rows):
+        current_row = populate_current_row(triangle, current_row_idx)
+        triangle.append(current_row)
+    return triangle
```

Comment:
```
      1
     1 1
    1 2 1
   1 3 3 1
  1 4 6 4 1
  ```
  Pascal's triangle would print out a triangle like this, instead of printing whole of the triangle in a single line.

Reply:
Added a print_pascal_triangle method to the implementation that prints out a triangular form of Pascal's triangle. I hope this helps.